### PR TITLE
fiona / add Basket indices page

### PR DESCRIPTION
**Pull request by fiona-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by habib-deriv — APPROVED** [review]
Looks good to me

File: src/pages/markets/basket/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+]
+
+const BasketPage = () => (
+    <Layout>
+        {/* <SEO
+            description={localize(
+                'Trade forex online with Deriv. Open a demo account and enjoy a wide range of currency pairs on our responsive and intuitive trading platforms.',
+            )}
+            title={localize('Forex trading online | Forex trading demo account | Deriv')}
+        /> */}
```
Comment (sean-binary) [inline src/pages/markets/basket/index.js:44]: Any reason this is commented out?

File: src/pages/markets/basket/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    {
+        header: <Localize translate_text="Withdraw" />,
+        text: (
+            <Localize translate_text="Conveniently withdraw your funds through any of our supported withdrawal methods." />
+        ),
+        icon: <img src={WithdrawIcon} alt="" width="32" height="32" />,
+    },
+]
+
+const BasketPage = () => (
```
Comment (sean-binary) [inline src/pages/markets/basket/index.js:37]: If possible, can we make all new JS files as TSX files if it has JSX in them and also declare their return types?
```suggestion
const BasketPage = (): ReactElement => (
```

File: src/pages/markets/components/sections/_nav-tab.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ import { Localize, LocalizedLink } from 'components/localization'
 import device from 'themes/device'
 
 const TabsContainer = styled(Flex)`
-    background-color: var(--color-grey-23);
+    background-color: rgba(249, 251, 255, 1);
 `
```
Comment (sean-binary) [inline src/pages/markets/components/sections/_nav-tab.js:11]: We can pass this as `background` into `Flex` directly.

File: src/pages/markets/basket/index.js (hunk 1 above)
Comment (fiona-deriv) [reply src/pages/markets/basket/index.js:44]: gonna update this with the basket info once provided

**Pull request closed without merging**